### PR TITLE
Translate index.mdx (to ES) and add Spanish models to quicktour.mdx examples

### PR DESCRIPTION
**Pull request by omarespejel**
# What does this PR do?
Related to issue #15947 
1. Translate `index.mdx` to Spanish
2. Replace English models and a dataset in `quicktour.mdx` with Spanish versions.

## Relevant
- Translation of `index.mdx` included the list of compatible models (not the papers´ and models´ names). Since this should not be updated manually, I can come back to the original text if required.
- Spanish models selected for quicktour are:
  - clasificador = pipeline('sentiment-analysis', model="pysentimiento/robertuito-sentiment-analysis")
  - reconocedor_de_voz = pipeline("automatic-speech-recognition", model="jonatasgrosman/wav2vec2-large-xlsr-53-spanish", device=0)
- Spanish ASR dataset selected is:
  - dataset = datasets.load_dataset("PolyAI/minds14", name="es-ES", split="train") 
 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

### CODE REVIEW

File: docs/source/es/quicktour.mdx [hunk 1]
```diff
@@ -80,24 +80,24 @@ Importa [`pipeline`] y especifica la tarea que deseas completar:
...
-Para más de un enunciado entrega una lista de frases al [`pipeline`] que devolverá una lista de diccionarios:
+Para más de un enunciado, entrega una lista al [`pipeline`] que devolverá una lista de diccionarios:
 
 ```py
->>> results = classifier(["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."])
->>> for result in results:
-...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
-label: POSITIVE, with score: 0.9998
-label: NEGATIVE, with score: 0.5309
+>>> resultados = clasificador(["Estamos muy felices de mostrarte la biblioteca de 🤗 Transformers.", ""Esperamos que odies lo mucho que te gusta 😈."])
```
Comment (kumar-abhishek) [inline docs/source/es/quicktour.mdx:96]: Looks like you have an extra double quote here:  `""Esperamos`  because of which build is failing.

Comment (omarespejel) [reply docs/source/es/quicktour.mdx]: Thank you @kumar-abhishek! That was the error! :)

Comment (kumar-abhishek) [reply docs/source/es/quicktour.mdx]: @omarespejel there is still a build failure, please fix that.

**Review by osanseviero — APPROVED** [review]
Nice!

**Review by sgugger — APPROVED** [review]
Thanks a lot for all of those! Just make sure to run `make style` on your branch to fix the quality issue.

**Comment by omarespejel** [comment]
@sgugger, I ran `make style` and made some manual debug but cannot get `check_code_quality` to pass. Any idea why could it be? 

I think the error is in[ these lines](https://github.com/huggingface/transformers/pull/16685/files#diff-517e793a1e18859abaf368b0c3f4d344231747ebc1ceb5154ca94159a7207bf1R110-R116):

`
A continuación, carga el dataset (ve 🤗 Datasets [Quick Start](https://huggingface.co/docs/datasets/quickstart.html) para más detalles) sobre el que quisieras iterar. Por ejemplo, vamos a cargar el dataset [MInDS-14](https://huggingface.co/datasets/PolyAI/minds14):

```py
>>> from datasets import load_dataset, Audio

>>> dataset = load_dataset("PolyAI/minds14", name="es-ES", split="train")  # doctest: +IGNORE_RESULT
```
`

**Comment by sgugger** [comment]
Make sure you have the latest version of `hf-doc-builder` installed (`pip install hf-doc-builder -U`) then run `make style`.

**Comment by omarespejel** [comment]
Sorry @sgugger, even with updating `hf-doc-builder` to the latest version, `0.4.0`,  and running `make style` the `check_code_quality` error continues appearing. I am not able to find the error and the [feedback in CircleCI](https://app.circleci.com/pipelines/github/huggingface/transformers/39893/workflows/35203812-3c00-42fa-b7e6-2d1789c21b12/jobs/449693) is not revealing (`ValueError: 1 files should be restyled!`).

When running `make style` I get this error (I tried to debug in other ways but the error continues):
Screen Shot 2022-05-11 at 23 45 35

In the meantime, I merged the PR with the error so we can get the index and quicktour before the release tomorrow. However, please let me know if we should proceed in another way. Sorry for this cumbersome merge.

**Pull request merged**